### ISSUE TITLE
Common Good & Public good writing

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#2277]
More than welcome I would like to fund bounty for writing about the common good and the public good. 
You can frame it within the case study as KodaDot is doing public well being open source and sharing knowledge further and growing "framework" for NFTs 
Feel free to include their MetaPrime as common/public good for experimental NFTs

We've talked about this with @cryptodamsky and I gues whoever would like to take this one, happy to fund it. 
I guess we aim for the long-read articles, short-read would be nice as well, this more likely will be continuos bounty as we are collecting more perspectives on the public good.

Work done on Internet should be recognized as a public good. 

Maybe is worth including ReFi, [like regenerative finance](https://twitter.com/carl_cervone/status/1450593675550019586)
### 
- https://guide.kusama.network/docs/learn-common-goods/
- https://www.youtube.com/results?search_query=public+goods
- https://www.youtube.com/watch?v=JcX5urijD0s - web3foundation
- https://twitter.com/owocki 
- https://twitter.com/gitcoin
- I've asked https://twitter.com/yangWao/status/1490978403230568448
- https://twitter.com/owocki/status/1384896939293093890
- https://twitter.com/lrettig/status/1118832963683270656
- https://twitter.com/gabriellemic/status/1228860215447048192
- https://twitter.com/_Dave__White_/status/1390686770900070404
- https://twitter.com/search?q=public%20good%20%40owocki&src=typeahead_click
- https://twitter.com/owocki/status/1373810510542426113
- https://twitter.com/VitalikButerin/status/1224905811274715138
- https://twitter.com/bianca_troveaux/status/1286336472409419777
- https://twitter.com/owocki/status/1474491893803196416
- https://twitter.com/carl_cervone/status/1450593675550019586
- https://greenpill.party/

**Comment by kkukelka** [comment]
I'm on it ✏️

**Comment by yangwao** [comment]
> > this more likely will be continuos bounty as we are collecting more perspectives on the public good
> 
> I'm currently pretty stretched, but I'd like to participate on this one if it's still going to be a thing in March. I'll do some reading in the meantime 😄

Sure, more perspectives matter.

**Comment by yangwao** [comment]
> I'm on it ✏️

Any draft out there? :)

**Comment by kkukelka** [comment]
> > I'm on it ✏️
> 
> Any draft out there? :)

I'll try to finish it up in the coming days. 👍

**Comment by yangwao** [comment]
> hi @kkukelka - reach out if you want to collaborate. I'm currently gathering articles and resources on public good, Re-Fi and governance; right now it's in a roam graph.
> 
> Not sure what type of form you want this work to take but it would be cool to build a research collective or a shared repo or journal somewhere.

Oh, I would happily participate on this page as a researcher as already did some readings on this. 
Seems awesome-public-good has room to grow https://github.com/search?q=awesome-public-good 😇

**Comment by cryptodamsky** [comment]
My 2 cents were submitted to KodaDot publication.. Will see, when or if the article sees the light of the world.

**Comment by kkukelka** [comment]
@yangwao just submitted my draft https://medium.com/kodadot/on-sustaining-open-source-as-a-public-good-a3e8c36e67d6
In this first article, I wanted to establish a foundation for the term "public good" in the context of the internet, web3, and kodadot. Furthermore I tried to convey the importance of knowledge in this regard and how we at kodadot help to sustain such knowledge in the open source ecosystem 👍

**Comment by yangwao** [comment]
We have meanwhile discussion on harvesting knowledge exchange on public good and quadratic funding https://github.com/kodadot/nft-gallery/discussions/2946

**Comment by yangwao** [comment]
Should be ongoing stuff as there are lot's of materials on Internet about it :)

**Comment by kkukelka** [comment]
> Should be ongoing stuff as there are lot's of materials on Internet about it :)

Yeah sorry I wasn't sure how to go about the payment for the issue. 😅

**Comment by GoodnessNwanebu** [comment]
Hi, is this still on? If yeah? What will you like to see in a "perfect" article?

**Comment by yangwao** [comment]
> Hi, is this still on?

yes :)

**Comment by poppyseedDev** [comment]
Update from Kusama docs:
Common Good (Parachain)
See [System Parachains](https://guide.kusama.network/docs/glossary/#system-parachains), which is generally preferred over the term "common good".
[ref](https://guide.kusama.network/docs/glossary/#common-good-parachain)

Checkout:
https://guide.kusama.network/docs/learn-system-chains/

**Comment by yangwao** [comment]
> See [System Parachains](https://guide.kusama.network/docs/glossary/#system-parachains), which is generally preferred over the term "common good".

yeah yet in web3 context it's different I guess. 

- https://en.wikipedia.org/wiki/Public_good_(economics)
- #2897

**Comment by iMac7** [comment]
@yangwao I've published an article on this but it's on my private medium account. Lmk if it's fine there or it should be on Kodadot's account.
Link - https://medium.com/@macharian77/web3-and-the-common-good-f73b06201095

@kkukelka great articles btw :clap:
